javalin/pom.xml

```diff
@@ -102,6 +106,13 @@
         <dependency>
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-unixsocket</artifactId>
+            <type>pom</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-unixsocket-server</artifactId>
+            <version>11.0.8</version>
```

Comment:
Is this related to anything? Should we extract version?

Reply:
~ https://github.com/javalin/javalin/blob/694fc80468834538d5cecf55b85f55e2e24b929b/javalin/src/test/java/io/javalin/TestUnixSocketConnector.kt

Reply:
So they split out an artifact for this? I think we should add it to the `dependencyManagement` section of the parent pom then, and use `jetty.version` rather than hardcoding.